venv.sh

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
```

Comment:
this is not needed. poetry will setup a venv for you.  use the poetry CLI commands to manage dependencies.